bindings/go/mio/mkv.go

```diff
@@ -50,6 +50,10 @@ func (mkv *Mkv) idxNew(id string) (err error) {
     if err != nil {
         return err
     }
+    if (mkv.idxID.u_hi >> 56) != 0x78 {
```

Comment:
Do you think we forcibly XORing the 0x78 will be better?
```
mkv.idxID.u_hi |= 0x7800000000000000;
```
Or maybe not.
"MSByte", what does it mean?

We need to say that, 0x78 is the type of object fid type.

Reply:
> forcibly XORing the 0x78 will be better?

Not sure. User may use this utility for debug purposes and use the same index in other apps, so better being clear about its fid.

MSByte is Most Significant Byte, see https://en.wikipedia.org/wiki/Bit_numbering#Most_significant_byte.

> need to say that, 0x78 is the type of object fid type.

May be. Wouldn't that be too much for the error msg?

Reply:
[Suggestion]
It would be better if code can be changed in a following way,

 if (m0_fid_tget(dix_fid) == m0_dix_fid_type.ft_id )

Reply:
Good idea, done.